.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav_v2.vtl

```diff
@@ -307,7 +310,7 @@
             </a>
           </li>
           <li>
-            <a href="/panthermail/">
+            <a href="https://www.chapman.edu/panthermail/">
```

Comment:
@mandythomas Just wondering, but why is this one the only one that's not `site://Chapman.edu/`? At first I thought it was because it didn't have any variables, but then I noticed other `href`s in this file that didn't have variables either. So I just wanted to understand.

Reply:
@kaylaziegler the /panthermail link is hardcoded with the domain because it's a 301 redirect setup in the IIS webservices on the WWW site and not a link to a real page in Cascade, And that same shortcut is not setup on other test or dev servers, it's only valid on WWW so we need to hardcode it in.